amazons3/amazons3_types.bal

```diff
@@ -28,38 +28,38 @@ public type AmazonS3Connector object {
     public string region;
 
     # Retrieve the existing buckets.
-    # + return - If success, returns BucketList object, else returns AmazonS3Error object
-    public function getBucketList() returns Bucket[]|AmazonS3Error;
+    # + return - If success, returns BucketList object, else returns error object
+    public function getBucketList() returns Bucket[]|error;
 
     # Create a bucket.
-    # + return - If success, returns Status object, else returns AmazonS3Error object
-    public function createBucket(string bucketName) returns Status|AmazonS3Error;
+    # + return - If success, returns Status object, else returns error object
+    public function createBucket(string bucketName) returns Status|error;
 
     # Retrieve the existing objects in a given bucket.
     # + bucketName - The name of the bucket
-    # + return - If success, returns S3Object[] object, else returns AmazonS3Error object
-    public function getAllObjects(string bucketName) returns S3Object[]|AmazonS3Error;
+    # + return - If success, returns S3Object[] object, else returns error object
```

Comment:
Just say returns error, not the error object, check all the other places

Reply:
Fixed.